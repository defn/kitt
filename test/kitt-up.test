#!/bin/bash

function main {
  kitt up --help
}

main "$@"
