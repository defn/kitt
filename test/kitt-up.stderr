USAGE: kitt up [flags] -- args

Calls docker-compose up to bring up services.

args: arguments to docker-compose

flags:
  -D,--[no]fg:  run in the foreground (default: false)
  -h,--help:  show this help (default: false)
